frontend/src/App.js

```diff
@@ -1,6 +1,10 @@
+import CheckInComponent from "./components/CheckInComponent"
 function App() {
   return (
-    <h1>NEW REACT APP YAY</h1>
+    <>
+      <CheckInComponent></CheckInComponent>
+    </>
```

Comment:
If you're returning a single component with no props you can reduce this to one line like `<CheckInComponent />` without wrapping it in `<></>`